File: src/WebClient/WareHouse/src/app/entity/WareHouseItemUnit.ts
import { BaseEntity } from "./BaseEntity";
import { Unit } from "./Unit";
import { WareHouseItem } from "./WareHouseItem";

export interface WareHouseItemUnit extends BaseEntity {
    itemId: string;
    unitId: string;
    unitName: string;
    convertRate: number;
    isPrimary: boolean | null;
    item: WareHouseItem;
    unit: Unit;
}

File: src/Services/WareHouse/WareHouse.Domain/Entity/WareHouseItemUnit.cs
using System;
using System.Collections.Generic;
using Share.BaseCore.BaseNop;

#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class WareHouseItemUnit:BaseEntity
    {
        public string ItemId { get; set; }
        public string UnitId { get; set; }
      //  public string UnitName { get; set; }
        public int ConvertRate { get; set; }
        public bool? IsPrimary { get; set; }

        public virtual WareHouseItem Item { get; set; }
        public virtual Unit Unit { get; set; }
    }
}
